Pod/Extensions/SSO/SKYAuthContainer+SSO.m

```diff
@@ -161,4 +164,29 @@ - (NSURL *)sso_genCallbackURL:(NSString *)scheme
     return [[NSURL alloc] initWithScheme:scheme host:host path:@"/auth_handler"];
 }
 
+#pragma mark - Custom Token
+
+- (void)loginWithCustomToken:(NSString *)customToken
+           completionHandler:(SKYContainerUserOperationActionCompletion)completionHandler
+{
+    SKYLoginCustomTokenOperation *op =
+        [SKYLoginCustomTokenOperation operationWithCustomToken:customToken];
+    op.loginCompletionBlock = ^(SKYRecord *user, SKYAccessToken *accessToken, NSError *error) {
+        if (error) {
+            if (completionHandler) {
+                completionHandler(user, error);
+            }
+            return;
+        }
+
+        [self.container.auth updateWithUser:user accessToken:accessToken];
+        [self.container.push registerDeviceCompletionHandler:nil];
```

Comment:
I moved `registerDeviceCompletionHandler` inside `updateWithUser` in the last PR. So we can remove it here. The other parts look good!

refs: https://github.com/SkygearIO/skygear-SDK-iOS/blob/3e58465b9d6d99b78c90b2004821423908cd1303/Pod/Classes/SKYAuthContainer.m#L187